src/components/ItemList.svelte

```diff
@@ -1,4 +1,5 @@
 <script>
+  import lunr from "lunr";
```

Comment:
Hi @wlach I need some help on this. Storyshot doesn't like this import (see [CircleCI failed test](https://app.circleci.com/pipelines/github/mozilla/glean-dictionary/2126/workflows/0d515da1-5b19-417c-861b-45162d688d41/jobs/4073)). 
Changing this to `import * as lunr from "lunr"` doesn't work (though it would appease storybook) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#import_an_entire_modules_contents

I'm not sure how to work around this 😕